docs/en/cloud/Google-Dataproc.md

```diff
@@ -41,16 +41,25 @@ When creating a Dataproc cluster, Alluxio can be installed using an
 ### Create a cluster
 
 There are several properties set as metadata labels which control the Alluxio Deployment. 
-* A required argument is the root UFS address configured using **alluxio_root_ufs_uri**.
+* A required argument is the root UFS address configured using **alluxio_root_ufs_uri**. 
+If value `LOCAL` is provided, hdfs launched by the current dataproc cluster will be used as Alluxio root UFS.
 * Properties must be specified using the metadata key **alluxio_site_properties** delimited using
 a semicolon (`;`).
 
 ```console
```

Comment:
maybe better to split into separate console blocks, otherwise someone may try to run both? especially people like me that don't read the comments carefully